src/uct/api/uct.h

```diff
@@ -437,6 +437,25 @@ enum {
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief   @anchor UCT_FLUSH_FLAGS Flags that control completion semantic.
+ * 
+ * These flags control completion semantic for @ref uct_ep_flush and
+ * @ref uct_iface_flush functions.
+ */
+enum {
+    UCT_FLUSH_FLAG_LOCAL   = 0,         /**< Guarantees flush of local
```

Comment:
for current backward compatibility (0 was a default value)
do you propose to change this everywhere in UCP and tests as well?